Babylon/commands/api/workspace/update.py

```diff
@@ -21,7 +21,7 @@
 @timing_decorator
 @require_platform_key("api_url")
 @pass_azure_token("csm_api")
-@argument("workspace_id", type=QueryType())
+@argument("workspace_id", type=QueryType(), default="%deploy%workspace_id")
```

Comment:
```suggestion
@argument("workspace_id", type=QueryType())
```
I prefer to be explicit here as well